modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -70,7 +70,16 @@ definition {
 	macro _createObjectDefinition {
 		Variables.assertDefined(parameterList = "${en_US_label},${name},${requiredStringFieldName},${en_US_plural_label}");
 
-		var portalURL = JSONCompany.getPortalURL();
+		if (isSet(virtualHost)) {
+			if (!(isSet(port))) {
```

Comment:
nice! 👏🏻

Reply:
I wonder if in the future we can make it more global so there's no need to add this much of code in different macros 🤔 Maybe it would be a good start for the PoshiForApiUtils.macro?